zb-db/src/test/java/io/camunda/zeebe/db/impl/rocksdb/ZeebeRocksDbFactoryTest.java

```diff
@@ -113,4 +114,51 @@ void shouldFailIfPropertiesDoesNotExist(final @TempDir File pathName) {
         .hasMessageContaining(
             "Expected to create column family options for RocksDB, but one or many values are undefined in the context of RocksDB");
   }
+
+  @Test
+  void shouldOpenSnapshotOnlyDb(final @TempDir File path, final @TempDir File tempDir)
```

Comment:
🔧 Could we add a test that it fails on other methods

Reply:
I thought about it, but then figured it was simple enough, and tbh the main important part here is that we can snapshot. I can add a test for other methods, but can't guarantee we'll keep this up to date in the future. Maybe it's enough to make sure you can't open transactions or create column families :smile: